package.json

```diff
@@ -22,7 +22,7 @@
     "lint-staged": "^10.1.3",
     "pretty-quick": "^2.0.1",
     "prettier": "1.19.1",
-    "prettier-plugin-solidity": "^1.0.0-alpha.46",
+    "prettier-plugin-solidity": "1.0.0-alpha.46",
```

Comment:
unrelated to changes but locking this prettier version because prettier `1.0.0-alpha.48` is now out which results in a bunch of linting changes to our code. If we enable higher versions we will need to lint all the contracts again.

Reply:
you're unlocking or locking? it looks like you're unlocking

Reply:
To my understanding this pins us to use exactly this package version.

Reply:
Doesn't the "^" pin it?

Reply:
https://semver.npmjs.com/

I stand corrected. Based on this, the "^" does not pin it.